documentation/modules/upgrading/ref-upgrade-kafka-versions.adoc

```diff
@@ -15,7 +16,7 @@ include::snip-kafka-versions.adoc[leveloffset=+1]
 .Message format version
```

Comment:
Should we add similar section for `inter.broker.protocol.version`? It is now refered in various parts of the upgrade guide.

Reply:
Nice suggestion. I added a few sentences about the inter-broker protocol version. Please review and add more accurate words if needed!